go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/xanzy/go-gitlab
+module github.com/iskanderbs/go-gitlab
```

Comment:
Uuh... Please don't change this 😏

Reply:
Sorry for my stupid solution, thank you for your reply)

Reply:
No worries, all good 👍